### ISSUE TITLE
Whether I can use single nuclei RNA-seq data as input

### ISSUE THREAD

**Issue by DexinYang1998** [Danko-Lab/BayesPrism#11]
I am wondering whether I can use single nuclei RNA-seq data as input. Compared with scRNA-seq, snRNA-seq tends to capture the mRNA inside the nuclei. I think this might introduce some potential biases.

**Comment by tinyi** [comment]
Yes. When using scNUC-seq as the input, please use the updated cell type
fraction as the batch effect is often very strong.
